pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesTest.java

```diff
@@ -252,6 +252,13 @@ public void testGetNamespaces() throws Exception {
         expectedList.sort(null);
         assertEquals(namespaces.getTenantNamespaces(this.testTenant), expectedList);
 
+        try {
+            // check the tenant name is valid
+            namespaces.getTenantNamespaces(this.testTenant + "/default");
+            fail("should have failed");
+        } catch (RestException e) {
```

Comment:
It's better to catch the specific RestException or check the response code.

Reply:
Done.